contents/blog/interview-snapshot-guide.md

```diff
@@ -0,0 +1,69 @@
+
+Championed by Teresa Torres in the book [Continuous Discovery Habits](https://www.amazon.co.uk/Continuous-Discovery-Habits-Discover-Products/dp/1736633309), interview snapshots are great conversation starters you can create with minimal effort – an 80/20 method that allows us to gather feedback, generate quick insights, build MVPs, measure their success and iterate. 
+
+In this guide, I'll share how we create user interview snapshots at PostHog – including the following template that you can download and fill out, or recreate in Google Docs, Figma or any other tool of your choice.
+
+// Annika to create this
+
+## Interview snapshots explained
+
+To create a user interview snapshot, you need a list of insights from a user interview. Ideally you take those notes during the interview, or you can ask one of your colleagues to do so. 
+
+While taking notes, focus on how the user is using your product, and how they feel while using it. What makes them excited? What frustrates them? Those are the insights that later go into your interview snapshot.
+
+Another approach is to record the conversation, transcribe and summarise it. This approach is more time consuming, so we don’t recommend it. Though you might want to record the conversation in addition to taking notes, if later on you are looking for word-by-word quotes.
+
+Once you have your insights available, you can open up your interview snapshot template and start filling it out. From experience, we recommend blocking 30 minutes after your interview to review your notes and fill out the user interview snapshot. Here is a step by step guide on how to fill it out:
+
+### 1. Note high-level information of your user
+
+To set the scene, add the user’s name at the top of the interview snapshot. For B2B companies, we also recommend adding the job role and company name. It’s also handy to add the date of the interview and a high-level summary what the interview was about. If available, add a picture of the user and the company logo, if the context is B2B. Believe us, this will spur your memory if you revisit the interview in a couple weeks’ time.
+
+### 2. Add any relevant background information
+
+What information about your user is relevant to your company and product? Their age? Their responsibilities in their job role? How their morning routine looks like? Write this information down in the “quick facts” column. We also recommend writing down any previous experience the user has with your product.
+
+### 3. List the opportunities you’ve identified
+
+Did the user express any pain points or desires in relation to your product or problem area? Write these down in the “Opportunities” column. During the interview, make sure you really get to the bottom of the user’s needs, and avoid the [XY problem](https://xyproblem.info/). Make sure to list problems here, and not feature ideas. 
```

Comment:
```suggestion
Did the user express any pain points or desires in relation to your product or problem area? Write these down in the “Opportunities” column. During the interview, make sure you really get to the bottom of the user’s needs, and avoid the [XY problem](https://xyproblem.info/). Make sure to list problems here, and **not feature ideas.** 
```

Is it worth doing a short boxout on the XY problem? Always nice to keep people on the page and it would add some useful context.